lib/auth/initial_seeder.rb

```diff
@@ -13,14 +17,11 @@ def initialize
 
   def gds_group_exists?
     begin
-      SelfService.service(:cognito_client).get_group(
-        group_name: TEAMS::GDS,
-        user_pool_id: Rails.configuration.cognito_user_pool_id
-      )
-    rescue Aws::CognitoIdentityProvider::Errors::ResourceNotFoundException
+      get_group(group_name: TEAMS::GDS)
+    rescue AuthenticationBackend::UserGroupNotFoundException
       Rails.logger.warn('The GDS group does not exist in Cognito!')
```

Comment:
if we're abstracting it completely...
```suggestion
      Rails.logger.warn('The GDS group does not exist in the authentication backend!')
```